apps/sertra/components/ServerDetail.module.css

```diff
@@ -0,0 +1,354 @@
+.container {
+  width: 100%;
+  display: flex;
+  column-gap: 2em;
+  flex-direction: column;
+}
+
+.headerRow {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.contentRow {
+  display: flex;
+  column-gap: 2em;
+}
+
+.columnLeft {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+}
+
+.columnRight {
+  display: flex;
+  flex-direction: column;
+}
+
+
+.descriptionContent,
+.row {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 0.75em 1em 12px 0.75em;
+  margin-bottom: 0.5em;
+  gap: 1em;
+  font-size: 16px;
+  line-height: 140%;
```

Comment:
The aforementioned recent PR also changed `line-height`s from percentages to unitless values, which is [recommended](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#prefer_unitless_numbers_for_line-height_values).